sdk/eventhub/Azure.Messaging.EventHubs.Processor/perf/Infrastructure/ProcessorTest.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Tests;
+using Azure.Storage.Blobs;
+using Azure.Test.Perf;
+
+namespace Azure.Messaging.EventHubs.Processor.Perf.Infrastructure
+{
+    public abstract class ProcessorTest<TOptions> : EventPerfTest<TOptions> where TOptions : ProcessorOptions
+    {
+        private readonly BlobContainerClient _checkpointStore;
+
+        protected EventProcessorClient EventProcessorClient { get; }
+
+        public ProcessorTest(TOptions options) : base(options)
+        {
+            _checkpointStore = new BlobContainerClient(GetEnvironmentVariable("STORAGE_CONNECTION_STRING"),
+                $"CheckpointStore-{Guid.NewGuid()}".ToLowerInvariant());
+
+            EventProcessorClient = new EventProcessorClient(
+                _checkpointStore,
+                EventHubConsumerClient.DefaultConsumerGroupName,
+                EventHubsTestEnvironment.Instance.EventHubsConnectionString,
+                EventHubsTestEnvironment.Instance.EventHubsNamespace,
+                new EventProcessorClientOptions()
+                {
+                    LoadBalancingStrategy = options.LoadBalancingStrategy,
+                });
+        }
+
+        public override async Task SetupAsync()
+        {
+            await base.SetupAsync();
+            await _checkpointStore.CreateIfNotExistsAsync();
```

Comment:
@jsquire: This comment was from a much older version of the PR.  Do you still think the EventHubs perf tests should use `EventHubScope/StorageScope`, or is it sufficient to use `EventHubsTestEnvironment/StorageTestEnvironment`?

Reply:
For consistency, it would be good to use the `StorageScope`, though it really doesn't buy you much in this case, since it would have to .    I'm good if you decide to skip it - you're already ensuring the container is created and deleted, which is all the scope really does.   _(the scope is really intended to be instantated instantiated in a single test method for an RAII-style management)_

The `StorageTestEnvironment` is the important part.